components/combobox/__tests__/combobox.browser-test.jsx

```diff
@@ -358,6 +358,70 @@ describe('SLDSCombobox', function() {
 				index: 0,
 			}).simulate('keydown', keyObjects.DELETE);
 		});
+
+		it('selects a menu item and scrolls when a letter key is pressed in read-only mode', () => {
+			wrapper = mount(<DemoComponent variant="readonly" />, {
+				attachTo: mountNode,
+			});
+			let nodes = getNodes({ wrapper });
+
+			nodes.input.simulate('click', {});
```

Comment:
can we change this to a down arrow instead of a click, so we can test the issue I just found about the down arrow even being scoped to an open menu?